lib/Dialect/FIRRTL/Import/FIRAnnotations.cpp

```diff
@@ -0,0 +1,182 @@
+//===- FIRAnnotations.cpp - .fir to FIRRTL dialect parser -----------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// Provide utilities related to dealing with Scala FIRRTL Compiler Annotations.
+//
+//===----------------------------------------------------------------------===//
+
+#include "circt/Dialect/FIRRTL/FIRAnnotations.h"
+
+#include "mlir/IR/BuiltinTypes.h"
+#include "mlir/IR/OperationSupport.h"
```

Comment:
I'd consider a "using json = llvm::json;" to simplify the qualifications below. 

"using namespace" things in headers are evil, but they are ok in .cpp files.

Reply:
Good idea. Fixed in the rebase.